library/sanitize.inc.php

```diff
@@ -167,3 +167,9 @@ function sanitizeNumber($number)
         return $clean_number;
     }
 }
+
+// Sanitized unserialize; goal it to remove all unserialize calls entirely from the codebase
```

Comment:
Instead of wrapping all these core function calls, we should just add the parameter in place. i.e `unserialize(object, ['allowed_classes' => false])` . Remember that every additional function calls add to slowing overall performance.
It's also my opinion that in 95% of serialize/unserialize use cases, the vulnerability is minimum to none depending how they are used. So, simply disallowing them to be used should not be the goal.

Reply:
hi @sjpadgett ,
Still a work in progress. This is more of an organization step for me as I take a look at all the use cases and still not sure where it will all end up. I'm now having fun obliterating postnuke :)